### PR TITLE
Set auto cleanup to 30 days and change eviction type

### PR DESCRIPTION
**Pull request by JeffreyDevloo**

### CODE REVIEW

File: ovs/lib/alba.py [hunk 1]
```diff
@@ -610,6 +605,80 @@ def add_cluster(alba_backend_guid, abm_cluster=None, nsm_clusters=None):
...
+        :rtype: NoneType
+        """
+        if not isinstance(days, int) or 0 > days:
+            raise ValueError('Number of days must be an integer > 0')
+        if config is None:
+            alba_backend = AlbaBackend(alba_backend_guid)
+            config = Configuration.get_configuration_path(key=alba_backend.abm_cluster.config_location)
+        if AlbaController.can_set_auto_cleanup():
+            # Check if the maintenance config was not set (disabled)
+            maintenance_config = AlbaCLI.run(command='get-maintenance-config', config=config)
```
Comment (simonOpenV) [inline ovs/lib/alba.py]: Any specific reason why you use the alba cli to fetch the config? below, you have function for this purpose

File: webapps/api/backend/views/albabackends.py [hunk 2]
```diff
@@ -409,3 +408,63 @@ def expand_nsm_clusters(self, albabackend, version, cluster_names=None, amount=0
...
+                                             error_description="'auto_cleanup_deleted_namespaces' should be a positive integer or 0")
+        return AlbaController.set_auto_cleanup(alba_backend_guid=albabackend.guid, days=days)
+
+    @link()
+    @log()
+    @required_roles(['read', 'write', 'manage'])
+    @return_simple()
+    @load(AlbaBackend, validator=_validate_access)
+    def get_maintenance_config(self, albabackend):
+        # type : (AlbaBackend, int) -> None
```
Comment (simonOpenV) [inline webapps/api/backend/views/albabackends.py]: None rtype?

Comment (simonOpenV) [reply webapps/api/backend/views/albabackends.py]: get_x returns a celery id? Is this appropriate naming? When does the Albacli return? does this return the config dict? if so, correct function name, wrong docstring. If not, wrong function name imho

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2016 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js]: 2018

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2016 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js]: 2018

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        self.formFieldMapping = ko.observable(fieldMapping || {});
+        self.formMapping = ko.observable(formMapping || {});
+        self.formMetadata = ko.observable(metadata || {});
+        self.confirmOnly = ko.observable(false);
+
+        // Computed
+        self.hasHelpText = ko.computed(function() {
+            var hasText = {};
+            $.each(self.formQuestions(), function(index, item) {
+                var key = 'alba:wizards.' + self.wizardName + '.gather.' + item().field() + '_help';
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js]: we have a .format for javascript too. (ovs/lib/generic.js)

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2017 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js]: Almost there, still wrong

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        var self = this;
+
+        // Variables
+        self.data   = stepOptions.data;
+        self.shared = shared;
+
+        // Computed
+        self.canContinue = ko.computed(function() {
+            if (self.data.loading()) {
+                return {value: false, reasons: [$.t('alba:wizards.' + self.data.wizardName + '.gather.loading')], fields: []}
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js]: Could be string-formatted properly too

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/index.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2016 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/viewmodels/wizards/editmaintenance/index.js]: ctrl+c ctrl-v my previous comment

File: webapps/frontend/app/views/wizards/editmaintenance/confirm.html [hunk 9]
```diff
@@ -0,0 +1,91 @@
+<!--
+Copyright (C) 2016 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/views/wizards/editmaintenance/confirm.html]: 2018

File: webapps/frontend/app/views/wizards/editmaintenance/gather.html [hunk 10]
```diff
@@ -0,0 +1,77 @@
+<!--
+Copyright (C) 2017 iNuron NV
```
Comment (simonOpenV) [inline webapps/frontend/app/views/wizards/editmaintenance/gather.html]: 2018

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js (hunk 5 above)
Comment (JeffreyDevloo) [reply webapps/frontend/app/viewmodels/wizards/editmaintenance/data.js]: They do the same thing ;)

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js (hunk 7 above)
Comment (JeffreyDevloo) [reply webapps/frontend/app/viewmodels/wizards/editmaintenance/gather.js]: Same result :)

File: webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js (hunk 3 above)
Comment (JeffreyDevloo) [reply webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js]: Why don't you spin up some code to adjust all the headers within the repo

Comment (wimpers) [reply webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js]: For new files the (C) notice the year is when the file is created, not when last updated!

Comment (delamonpansie) [reply webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js]: vi crontab
0 0 1 1 *: find ~repo/framework | xargs sed -i s/$(($(date +%Y) - 1))/$(date +%Y)/; git commit -am "copyright"; git push

**Pull request merged**